libraries/ms-drivers/src/adt7476a_fan_controller.c

```diff
@@ -2,6 +2,168 @@
 #include <stddef.h>
 #include <string.h>
 
+#ifdef MPXE
+#include <stdlib.h>
+
+#include "adt7476a.pb-c.h"
+#include "adt7476a_fan_controller.h"
+#include "adt7476a_fan_controller_defs.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "i2c.h"
+#include "interrupt.h"
+#include "soft_timer.h"
+#include "store.h"
+#include "stores.pb-c.h"
+
+static MxAdt7476aStore s_store = MX_ADT7476A_STORE__INIT;
+static uint32_t s_speed[NUM_ADT_PWM_PORTS] = { 0 };
+static bool s_status[NUM_ADT_PWM_PORTS] = { 0 };
+
+static void prv_update_store() {
```

Comment:
this should be `prv_export_store()`, I guess just by convention we've been going with 'export' means push something to python, whereas 'update' means read something from python and update the local value